src/table/body-cell/index.tsx

```diff
@@ -11,6 +11,8 @@ interface TableBodyCellProps {
   wrapLines: boolean | undefined;
   isFirstRow: boolean;
   isLastRow: boolean;
+  isEvenRow?: boolean;
+  stripedRows: boolean | undefined;
```

Comment:
```suggestion
  stripedRows?: boolean;
```
Lets keep this consistent with the previous line